src/components/MapboxMap.tsx

```diff
@@ -74,7 +74,7 @@ export interface MapboxMapProps<T> {
  * `import 'mapbox-gl/dist/mapbox-gl.css';`
  *
  * Or, user may add a stylesheet link in their html page:
- * `<link href="https://api.mapbox.com/mapbox-gl-js/VERSION/mapbox-gl.css" rel="stylesheet">`
+ * `<link href="https://api.mapbox.com/mapbox-gl-js/v2.9.2/mapbox-gl.css" rel="stylesheet" />`
```

Comment:
is there a reason this is changed to a specific version?

Reply:
it's the version that we currently have in package.json so I think we should just specify it here to match it

Reply:
I'm imagining it might be a little weird then if we bump to a new minor/patch but forget to update here. our docs would essentially be telling people to use an older version. but, if that seems unlikely, then I'm fine with leaving it in

Reply:
I think leaving it as VERSION seems a bit vague and we do ant people to adhere to the same version we use in package.json to avoid potential version conflict, so we just need to be mindful about updating docs when we bump mapbox version.